rfcs/01924_tls_attestation_ca.md

```diff
@@ -0,0 +1,161 @@
+# RFC#01924: Remote Attestation via TLS (Certificate Authority)
+
+## Objective
+
+Remotely attest applications in Trusted Execution Environments (TEEs) over the
+TLS protocol.
+
+## Background
+
+A lot of existing applications that would benefit from the remote attestation
+use **TLS protocol** to create secure channels with their clients. But
+integrating the remote attestation might bring up unnecessary complications in a
+form of completely rewriting the application connection logic.
+
+Thus, instead of refactoring existing applications and implementing a new
+protocol, we propose to integrate remote attestation into TLS by
+cryptographically binding **TLS certificates** to the **TEE firmware**.
+
+## Overview
+
+In order to allow clients to use pure TLS to connect to TEE applications we
+propose a Proxy Attestation Service.
+
+Proxy Attestation Service - is a service that can be used by clients to remotely
+attest TEE applications. The key idea is: instead of attesting TEE applications
+themselves, clients attest the proxy, which attests TEE applications on behalf
```

Comment:
Which proxy? I thought the point is to have a CA, so maybe we should call it CA?